ext/standard/http.c

```diff
@@ -360,3 +361,27 @@ PHP_FUNCTION(request_parse_body)
 	SG(request_parse_body_context).throw_exceptions = false;
 	memset(&SG(request_parse_body_context).options_cache, 0, sizeof(SG(request_parse_body_context).options_cache));
 }
+
+PHP_FUNCTION(http_get_last_response_headers)
+{
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	if (!Z_ISUNDEF(BG(last_http_headers))) {
+		RETURN_COPY_VALUE(return_value, &BG(last_http_headers));
+	} else {
+		RETURN_NULL();
+	}
+}
+
+PHP_FUNCTION(http_clear_last_response_headers)
+{
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	zval_ptr_dtor(&BG(last_http_headers));
+	ZVAL_UNDEF(&BG(last_http_headers));
+	RETURN_NULL();
```

Comment:
Strictly speaking the engine initializes it to NULL for you, so this is not necessary but eh... better be explicit ig.

Reply:
Actually this function returns void so RETURN_NULL is confusing here.